### PR TITLE
Vehicle Lock: add events

### PR DESCRIPTION
**Pull request by BrettMayson**
**When merged this pull request will:**
- As requested at: https://github.com/acemod/ACE3/issues/3594#issuecomment-699551537

### CODE REVIEW

File: addons/vehiclelock/functions/fnc_lockpick.sqf [hunk 1]
```diff
@@ -53,10 +53,12 @@ switch (_funcType) do {
         _returnValue = !([_unit, _veh] call FUNC(hasKeyForVehicle)) && {(locked _veh) in [2, 3]};
     };
     case "startLockpick": {
+        [QGVAR(start), [_veh]] call CBA_fnc_localEvent;
```
Comment (mharis001) [inline addons/vehiclelock/functions/fnc_lockpick.sqf:56]:
```suggestion
        [QGVAR(started), [_veh]] call CBA_fnc_localEvent;
```
To keep same tense as "finished" one.

**Review by mharis001 — COMMENTED** [review]
Should add to documentation.

File: addons/vehiclelock/functions/fnc_lockpick.sqf [hunk 2]
```diff
@@ -53,10 +53,12 @@ switch (_funcType) do {
         _returnValue = !([_unit, _veh] call FUNC(hasKeyForVehicle)) && {(locked _veh) in [2, 3]};
     };
     case "startLockpick": {
+        [QGVAR(started), [_veh]] call CBA_fnc_localEvent;
```
Comment (PabstMirror) [inline addons/vehiclelock/functions/fnc_lockpick.sqf:56]:
```suggestion
        [QGVAR(startedLockpick), [_veh]] call CBA_fnc_localEvent;
```

File: addons/vehiclelock/functions/fnc_lockpick.sqf [hunk 3]
```diff
@@ -53,10 +53,12 @@ switch (_funcType) do {
...
         [_vehLockpickStrength, [_unit, _veh, "finishLockpick"], {(_this select 0) call FUNC(lockpick)}, {}, (localize LSTRING(Action_LockpickInUse)), _condition, ["isNotInside", "isNotSwimming"]] call EFUNC(common,progressBar);
     };
     case "finishLockpick": {
         [QGVAR(setVehicleLock), [_veh, false], [_veh]] call CBA_fnc_targetEvent;
+        [QGVAR(finished), [_veh]] call CBA_fnc_localEvent;
```
Comment (PabstMirror) [inline addons/vehiclelock/functions/fnc_lockpick.sqf:61]:
```suggestion
        [QGVAR(finishedLockpick), [_veh]] call CBA_fnc_localEvent;
```

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**